### ISSUE TITLE
Wrong rendering of (at least the) rails in Nürnberg

### ISSUE THREAD

**Issue by versat** [tordanik/OSM2World#105]
http://maps.osm2world.org/?h=128&view=N&zoom=18&lat=49.41453&lon=11.14058&layers=B0TTFT

![image](https://cloud.githubusercontent.com/assets/7264306/12019347/5885012c-ad6f-11e5-9df6-cb852f58722d.png)

**Comment by tordanik** [comment]
Can you describe the issue in words? I'm not sure I'm seeing it. Do you mean the vertical bars in the bottom right?

**Comment by versat** [comment]
Yes i mean these vertical bars. And the background color is not the same (light green) for the area in the middle.
